src/platform/APM32/platform_mcu.h

```diff
@@ -172,3 +172,8 @@
 #define IO_CONFIG_GET_PULL(cfg)  (((cfg) >> 5) & 0x03)
 
 #endif
+
+#if defined(APM32F4)
+#define FLASH_CONFIG_STREAMER_BUFFER_SIZE 4   // Flash word = 32-bits
```

Comment:
Maybe you can default FLASH_CONFIG_STREAMER_BUFFER_SIZE  to `sizeof(FLASH_CONFIG_BUFFER_ALIGN_TYPE)`

Reply:
Done